app/views/checkouts/_checkouts_table.html.erb

```diff
@@ -29,9 +29,9 @@
             <td>
               <%= render BibItemComponent, bibitem: checkout %>
             </td>
-            <td class="renewal_count"><%= checkout.renewal_count if checkout.renewal_count.positive? %></td>
-            <td class="due_date"><%= render partial: 'checkouts/due_date', locals: { checkout: checkout } %></td>
-            <td class="status"><%= checkout.status_human %></td>
+            <td class="renewal_count"><span><%= checkout.renewal_count if checkout.renewal_count.positive? %></span></td>
```

Comment:
just curious why needed a `span` here, I am sure there is a reason just trying to understand

Reply:
ok makes sense, thanks :)